src/main/java/com/shippo/net/APIResource.java

```diff
@@ -401,11 +401,20 @@ else if (method.equals(RequestMethod.POST) || method.equals(RequestMethod.PUT))
 
 			String rBody;
 			Map<String, List<String>> headers;
-
+			
 			if (rCode >= 200 && rCode < 300) {
 				rBody = getResponseBody(conn.getInputStream());
-			} else {
+			}
+			else if(rCode == 429){
```

Comment:
I think you should separate the bugfix from the CI PR, just to keep the PR focused on one purposed at a time, that's usually better

Reply:
I will make sure to do that